src/Nethermind/Nethermind.Runner/Nethermind.Runner.csproj

```diff
@@ -92,6 +92,10 @@
     <None Remove="keystore\**" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="configs\holesky.cfg" />
```

Comment:
Why is that in there? Won't it remove holesky config on build? If is needed then what about holesky_archive?